vagrant/provisioning/roles/upgrade-solr/tasks/main.yml

```diff
@@ -1,3 +1,13 @@
+- name: Check systemd zookeeper file exists
```

Comment:
Minor - there's a more idiomatic way to see if a service exists, without depending on the specific location of the service file.  I have this in the AWS installer branch... notice how the `service_facts` is just empty, it just populates standard Ansible facts:

```yaml
- name: check status of fapolicyd service
  service_facts:

- name: some other task that should run when fapolicyd is installed
  ...
  when: ansible_facts.services["fapolicyd.service"] is defined
```

Reply:
... only small problem is that `service_facts` takes some few seconds to run.